include/boost/astronomy/coordinate/coord_sys/equatorial_coord.hpp

```diff
@@ -0,0 +1,194 @@
+#ifndef BOOST_ASTRONOMY_EQUATORIAL_COORD_HPP
+#define BOOST_ASTRONOMY_EQUATORIAL_COORD_HPP
+
+#include <iostream>
+#include <boost/units/io.hpp>
+#include <boost/geometry/core/cs.hpp>
+#include <boost/geometry/geometries/point.hpp>
+#include <boost/units/systems/angle/degrees.hpp>
+#include <boost/units/systems/si/plane_angle.hpp>
+#include <boost/astronomy/coordinate/coord_sys/coord_sys.hpp>
+
+namespace boost {
+    namespace astronomy {
+        namespace coordinate {
+
+            namespace bu = boost::units;
+            namespace bg = boost::geometry;
+            namespace bud = boost::units::degree;
+
+            //Right Ascension
+            template
+                    <
+                            typename CoordinateType = double,
+                            typename RightAscensionQuantity = bu::quantity<bu::si::plane_angle, CoordinateType>
+                    >
+            struct RightAscension {
+            private:
+                RightAscensionQuantity ra;
+            public:
+                RightAscension(){
+                    ra = 0.0 * bu::si::radian;
+                };
+
+                RightAscension(RightAscensionQuantity const& _ra) : ra(_ra) {}
```

Comment:
variable exposed to the user should more readable so in this case internal variable should be `_ra` and parameter should be `ra`

make this change to all the subsequent similar cases